setup.cfg

```diff
@@ -9,17 +9,6 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
```

Comment:
Sorry for the noise, I see you have done something like this. The end goal would be to make sure we have some coverall status in this PR.